### DISCUSSION TITLE
Ask about how to calculate pfJ in MTEX

### DISCUSSION THREAD

**Discussion by ErkangQiu** [mtex-toolbox/mtex#1764] [Ask Anything]
Hi MTEX group,

I am a PhD student major in structural geology. Recently, I used the MTEX to calculate fabric strength index to explore some problem about strain variation in a shear zone. As the most powerful tool to extract information from EBSD data, researchers could calculate some   fabric strength index (e.g. Jodf, M-index, Jmdf) directly by MTEX. However, it looks like that it will be more complex to get the J-index of pole figures (pfJ) by MTEX. Therefore, I want to ask about how to calculate pfJ in MTEX. I will be grateful if you could provide some help when you are available.
Thanks for your time!

Erkang Qiu

**Comment by lfgmorales** [comment]
<https://gist.github.com/kilir/8decff039ec5d59419d67c8adc1a61da>
[gist-og-image.png]
calculate pfJ using S2Fun or manual calulation, the latter might be not as exact but much faster #mtexScript<https://gist.github.com/kilir/8decff039ec5d59419d67c8adc1a61da>
gist.github.com<https://gist.github.com/kilir/8decff039ec5d59419d67c8adc1a61da>




Cheers

Luiz

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 12 Mar 2023, at 21:14, ErkangQiu ***@***.***> wrote:

﻿

Hi MTEX group,

I am a PhD student major in structural geology. Recently, I used the MTEX to calculate fabric strength index to explore some problem about strain variation in a shear zone. As the most powerful tool to extract information from EBSD data, researchers could calculate some fabric strength index (e.g. Jodf, M-index, Jmdf) directly by MTEX. However, it looks like that it will be more complex to get the J-index of pole figures (pfJ) by MTEX. Therefore, I want to ask about how to calculate pfJ in MTEX. I will be grateful if you could provide some help when you are available.
Thanks for your time!

Erkang Qiu

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1764>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5INZNGS24APWS3YVMVATW3Y4CXANCNFSM6AAAAAAVYJ562A>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1764", "url": "https://github.com/mtex-toolbox/mtex/discussions/1764", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by ErkangQiu** [reply]
Thanks Luiz!
